src/transformers/models/cohere/configuration_cohere.py

```diff
@@ -17,64 +18,59 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Cohere model configuration"""
-
 from ...configuration_utils import PretrainedConfig
-from ...utils import logging
-
-
-logger = logging.get_logger(__name__)
 
 
 class CohereConfig(PretrainedConfig):
     r"""
```

Comment:
When inheriting from `GemmaConfig` we end up with many extra new attributes - e.g. `head_dim` etc. perhaps we should keep the old logic of re-writing the whole config class as proposed in https://github.com/huggingface/transformers/pull/31211 initially .. WDYT?